learnwithpeople/settings.py

```diff
@@ -164,6 +164,7 @@
 
 DEFAULT_FROM_EMAIL = os.environ.get('DEFAULT_FROM_EMAIL', 'webmaster@localhost')
 SERVER_EMAIL = os.environ.get('SERVER_EMAIL', 'no-reply@p2pu.org')
+REPLY_TO_EMAIL = os.environ.get('REPLY_TO_EMAIL', 'learningcircles@p2pu.org')
```

Comment:
For this you can just use the DEFAULT_FROM_EMAIL - currently it's the same as REPLY_TO_EMAIL when deployed